models/vrp.rb

```diff
@@ -220,9 +220,18 @@ def self.check_consistency(hash)
       end
 
       # number of visits consistency
-      if !configuration[:schedule]
+      if configuration[:schedule]
+        if configuration[:schedule][:range_indices][:start] > 6
+          raise OptimizerWrapper::DiscordantProblemError.new('Schedule start index should be less or equal to 6')
+        end
+
+        if configuration[:schedule][:range_indices][:start] > configuration[:schedule][:range_indices][:end]
+          raise OptimizerWrapper::DiscordantProblemError.new('Schedule start index should be less than end')
```

Comment:
less than or equal to the end index?

Reply:
If it is equal then it makes no sense to provide a schedule.

Reply:
I meant your message is not in accord with your `if` condition! 

I don't want to get into an argument but I don't understand this "makes no sense" argument. If scheduling code can handle scheduling 2-days, it should be able to handle 1-day without any issues -- which is just calling OR-tools. It making no sense doesn't mean that our code shouldn't be generic. Generic code is easier to maintain, expand and plug/play with other code.

Reply:
You are right, my message and error are not consistent. And we can deal with the case where start and end are equal. It is just not intuitive to me. But you are right, it is better if we want to avoid throwing useless errors.